src/ui/public/vislib/__tests__/visualizations/tile_maps/markers.js

```diff
@@ -301,7 +301,7 @@ describe('Marker Tests', function () {
           var arr = markerLayer._dataToHeatArray(max);
           var index = _.random(mapData.features.length - 1);
           var feature = mapData.features[index];
-          var featureValue = parseInt(feature.properties.value / max * 100);
+          var featureValue = parseInt(feature.properties.value / max);
```

Comment:
I think these parseInt()s needs to be ditched.  They were removed in the original PR.